### DISCUSSION TITLE
Aqara smoke detector JY-GZ-03AQ

### DISCUSSION THREAD

**Discussion by brjhaverkamp** [Koenkk/zigbee2mqtt#19059] [Devices]
Hi all,

I have a couple of Aqara zigbee smoke detectors (JY-GZ-03AQ). I have connected them in z2m and now want to make sure they all sound the alarm when one detects smoke

I got the "sound-the-alarm" part working via linkage_alarm_state. If that variable is true, I can send a buzzer="alarm" to all other detectors.

However, I also want to "mute-them-all" when I press the button on one of them. (I don't want to run through the whole house muting all detectors individually). This part I haven't been able to get working yet. What exposed variable/property indicates that a button has been pressed on these devices?
 I have been trying to use buzzer_manual_mute. But that doesn't seem to change when the buzzer is muted manually (via pressing the button) Actually I am not even sure if "manually" indicates "via an MQTT message as opposite to actual smoke detected" or via a "manual button press as opposite to an automated trigger(mqtt or smoke)"

I am trying to write a function in node red to connect everything. But sofar only the sounding of the alarms works. Any hint  on how to get them all muted again is appreciated.

Kind regards,

Bert

Regards,

Bert

**Comment by Jens-Wymeersch** [comment]
@brjhaverkamp Hi Bert, 
Just wondering if you were able to get this sorted out ? And do the JY-GZ-03AQ work well with Z2M ?

**Comment by Jan1503** [comment]
I just bought a pair of these devices to test and eventually replace my nearly 10-years old Ei 650 detectors.
Since they are 'officially' usable in EU now I was hoping to find a smart replacement which tells me some stats and most importantly where the smoke was detected first. I have around 30 Ei devices installed which should be replaced by 06/25.
Being able to mute the new aqaras via mqtt is crucial but also have to find out how reliable these devices are...

**Comment by dagobert** [comment]
Have a look at this [Home Assistant blueprint](https://community.home-assistant.io/t/aqara-smoke-detector-linkage-alarm/517656) and specially the thread about it. I think these detectors cannot link directly :( but the linkage is always implemented by the Hub - even using only Aqara stuff. 
Still you seem to have to enable linkage_alarm.

**Comment by brjhaverkamp** [comment]
True, you have to enable linkage_alarm to get the linkage_alarm_state message when smoke is detected.  I use that in Node Red to then trigger the other smoke alarms. 

Unfortunately, it takes quite a bit of logic to be able to mute the alarms again from any of the other alarm. 
On the original alarming device, where the smoke was detected,  when pushing mute,  "buzzer_manual_mute" becomes TRUE
But it is harder to detect if the mute button is pressed on any of the other "secondary" smoke alarms. When you press the button, a message is fired, but with no indication that the button is pressed :-( It would be so nice if there was a button_pressed object or similar..

**Comment by Umigarsik** [comment]
Fire prevention starts with having the right tools in place, from extinguishers to alarm systems. A monitored fire alarm can detect danger early and notify emergency responders instantly. If you’re looking for one, here’s a good source [fire protection company toronto](https://www.herbertwilliams.com/). This technology is especially useful for businesses and multi-unit buildings where immediate action is critical. It’s a small investment that can save lives.
